src/components/Footer/RenderCompanyInfoSection.jsx

```diff
@@ -23,6 +23,20 @@ const TwitterIcon = () => (
   </SvgIcon>
 );
 
+// Linkedin Icon
```

Comment:
Could we just import the built-in linkedin icon from the material ui library instead?

https://mui.com/material-ui/material-icons/?query=linked&selected=LinkedIn

We only had to do this for twitter because it didn't have the updated X icon version.